scripts/publish.sh

```diff
@@ -235,6 +243,8 @@ sync_latest_image() {
 	region=${1}
 	account_id=${2}
 
+	get_latest_dockerhub_version ${region}
```

Comment:
We can't call this function every time this script is used since it will require secrets manager permissions. So now we only call it when needed.